src/views/v2/day/type-separator.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * View: Day View Type separator
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/day/type-separator.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+
```

Comment:
* :no_entry_sign: Missing @package tag in file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: f002fc7d5b20db8ee6e01d5e31a02c0e</sub>